src/main/java/net/starschema/clouddb/jdbc/BQSupportFuncts.java

```diff
@@ -370,7 +370,7 @@ public static String logAndGetQueryState(Job pollJob) throws IOException {
             + " ; "
             + pollJob.getJobReference().getJobId()
             + " ; "
-            + (System.currentTimeMillis() - pollJob.getStatistics().getStartTime()));
+            + (System.currentTimeMillis() - pollJob.getStatistics().getCreationTime()));
```

Comment:
The original issue came about because we didn't check `getStartTime()` is not null or empty in the check above. 

Although I don't think it is possible for `getCreationTime()` to be null or empty in this case, I think we can go a bit further in this fix by wrapping it in an `Optional`?

Something like:

```java
Optional.ofNullable(pollJob.getStatistics().getCreationTime()).ifPresent(...);
```

Reply:
I added a commit that addresses this. Let me know if you want me to alter it. Thanks!